docs/cover.md

```diff
@@ -23,19 +32,30 @@ To install Mapbox Navigation using [CocoaPods](https://cocoapods.org/):
 
 Alternatively, to install Mapbox Navigation using [Carthage](https://github.com/Carthage/Carthage/):
 
-1. Create a [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#binary-only-frameworks) with the following dependency:
-   ```cartfile
-   github "mapbox/mapbox-navigation-ios" ~> ${MINOR_VERSION}
+1. Go to your [Mapbox account dashboard](https://account.mapbox.com/) and create an access token that has the `DOWNLOADS:READ` scope. **PLEASE NOTE: This is not the same as your production Mapbox API token. Make sure to keep it private and do not insert it into any Info.plist file.** Create a file named `.netrc` in your home directory if it doesn’t already exist, then add the following lines to the end of the file:
+   ```
+   machine api.mapbox.com
+     login mapbox
+     password PRIVATE_MAPBOX_API_TOKEN
    ```
+   where _PRIVATE_MAPBOX_API_TOKEN_ is your Mapbox API token with the `DOWNLOADS:READ` scope. 
 
-1. Run `carthage update --platform iOS` to build just the iOS dependencies.
+1. _(Optional)_ Clear your Carthage caches:
+   ```bash
+   rm -rf ~/Library/Caches/carthage/ ~/Library/Caches/org.carthage.CarthageKit/binaries/{MapboxAccounts,MapboxNavigationNative,mapbox-ios-sdk-dynamic}
+   ```
 
-1. Follow the rest of [Carthage’s iOS integration instructions](https://github.com/Carthage/Carthage#if-youre-building-for-ios-tvos-or-watchos). Your application target’s Embedded Frameworks should include `MapboxNavigation.framework`, `MapboxCoreNavigation.framework`, and `MapboxNavigationNative.framework`.
+1. Create a [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#github-repositories) with the following dependency:
+   ```cartfile
+   github "mapbox/mapbox-navigation-ios" "${MINOR_VERSION}"
+   ```
 
 ## Configuration
 
 1. Mapbox APIs and vector tiles require a Mapbox account and API access token. In the project editor, select the application target, then go to the Info tab. Under the “Custom iOS Target Properties” section, set `MGLMapboxAccessToken` to your access token. You can obtain an access token from the [Mapbox account page](https://account.mapbox.com/access-tokens/).
 
+1. By default, MAU billing is enabled so that SDKs usage of Mapbox APIs is [billed](https://www.mapbox.com/pricing/) together based on [monthly active users](https://docs.mapbox.com/help/glossary/monthly-active-users/) rather than individually by HTTP request. If you prefer to use request-based billing, set the `MBXNavigationBillingMethod` key in Info.plist to `request`
```

Comment:
I think we decided in https://github.com/mapbox/mapbox-navigation-ios/pull/2151#issuecomment-587857469 not to document this key but to document other supported Info.plist keys instead.

Reply:
Removed